site/docs/numbers/lookupNumbers.mdx

```diff
@@ -34,10 +35,7 @@ Currently, this service supports lookups of telephone numbers in the mainland Un
 
 ## Submit Number Lookup Order
```

Comment:
Can we change the use of "order" to "request" in all instances in this document?